samples/simpleSubOrchestrator/README.md

```diff
@@ -0,0 +1,46 @@
+# Sub Orchestrator sample
+
+## Pre-requisites
+
+You'll need core tools version v4.0.5095+. Please run `func --version` to ensure your core tools version is compatible.
```

Comment:
Should we add a link to where the user can install the Core Tools from? E.g., https://github.com/Azure/azure-functions-core-tools#installing ?

Reply:
Good idea, added in my latest commit